src/components/Clock/Clock.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Clock extends React.Component {
+  state = {
+    time: new Date(),
+  };
+
+  componentDidMount() {
+    this.timerId = setInterval(() => {
+      this.setState({
+        time: new Date(),
+      });
+
+      // eslint-disable-next-line
+      console.log(this.state.time.toLocaleTimeString());
+    }, 1000);
+  }
+
+  componentDidUpdate({ name: oldName }) {
+    const { name } = this.props;
+
+    if (oldName !== name) {
+      // eslint-disable-next-line
+      console.log(`The Clock was renamed from ${oldName} to ${name}`);
+    }
+  }
```

Comment:
You could just console.log it right directly when you changing the name and get rid of componentDidUpdate